workspaces/default/themes/base/partials/service-catalog/sidebar.html

```diff
@@ -41,16 +41,18 @@ <h1>{{ l("catalog_categories_header_text", "Category Filter") }}</h1>
     });
 
     var categories = {};
-    $CATALOG_TAG_LINKS.each(function (index, element) {
-      var tag = $(element).data("tag");
-      var countElement = $(element).find(".tag-link--count");
+    tagLinkElements.forEach(function (element) {
+      var tag = element.dataset.tag;
+      var countElement = element.querySelector('.tag-link--count');
 
       categories[tag] = {
         tag: tag,
         element: element,
         countElement: countElement,
-        count: parseInt(countElement.text(), 10),
+        count: parseInt(countElement.innerText, 10),
```

Comment:
shouldn't it be better `textContent` ?

Reply:
textContent may return text content with spacing and I didn't really want that

Reply:
good point